crates/tx/src/data/mod.rs

```diff
@@ -481,8 +481,6 @@ pub struct VpsResult {
     pub accepted_vps: BTreeSet<Address>,
     /// The addresses whose VPs rejected the transaction
     pub rejected_vps: BTreeSet<Address>,
-    /// The total gas used by all the VPs
-    pub gas_used: VpsGas,
```

Comment:
I've removed this because I believe it carries little information to the final user (since this type ends up being published in the tx result event) given that it only mentions the amount of gas used by vps without specifying the distribution across the different vps and inside each vp (compilation, execution, host Fns,...)